doozer/doozerlib/cli/scan_osh.py

```diff
@@ -345,23 +345,10 @@ def generate_commands(self, nvrs) -> Optional[list]:
         """
         cmds = []
 
-        cmd_template = "osh-cli mock-build --config={config} --profile custom-ocp --brew-build {nvr} --nowait"
         for nvr in nvrs:
-            if "-container" in nvr:
-                cmds.append(cmd_template.format(config="cspodman", nvr=nvr))
-
-            else:
-                if "el7" in nvr:
-                    rhel_version = 7
-                elif "el8" in nvr:
-                    rhel_version = 8
-                elif "el9" in nvr or nvr.startswith("rhcos"):
-                    rhel_version = 9
-                else:
-                    self.runtime.logger.error("Invalid RHEL version")
-                    raise Exception("Invalid RHEL Version")
-
-                cmds.append(cmd_template.format(config=f"rhel-{rhel_version}-x86_64", nvr=nvr))
+            cmd = f"osh-cli mock-build --config=auto --profile custom-ocp --brew-build {nvr} --nowait"
```

Comment:
Please use `--nvr` instead of `--brew-build`, which is deprecated:
```
% osh-cli mock-build --help | grep brew
  --brew-build=NVR               DEPRECATED alias for --nvr
```

Reply:
Done!